tests/configs/cjs/tables.js

```diff
@@ -0,0 +1,32 @@
+module.exports = [
```

Comment:
You should probably name this `.cjs` to actually test the CJS feature?

Reply:
Ok
I don't really understand how the tests are being run
Eslint is yelling at me:
```
Parsing error: "parserOptions.project" has been set for @typescript-eslint/parser.
The file does not match your project config: tests/configs/cjs/jest-dynalite-config.cjs.
The file must be included in at least one of the projects provided.
```
I couldn't find what to change to make it happy

Reply:
You need to add "*.cjs" to the `tsconfig.esllint.json` at the root of the repo. Atm I think it only includes `.js` and `.ts`

Reply:
>I don't really understand how the tests are being run
If you run `yarn test` you will see, they are effectively integration tests which ensure dynalite functions with an actual jest setup rather than unit.

The idea here is we want to ensure that `.cjs` configs do actually work and don't regress